sqlalchemy_media/stores/gcp.py

```diff
@@ -0,0 +1,94 @@
+""""
+    Google cloud storage
+"""
+from io import BytesIO
+
+from google.cloud import storage
+from google.cloud.exceptions import NotFound
+
+from sqlalchemy_media.exceptions import GCPError
+from sqlalchemy_media.optionals import ensure_gcs
+from .base import Store
+from ..typing_ import FileLike
+
+
+class GCPStore(Store):
+    """
+    Google Cloud Storage Implements Store base.
+    """
+
+    def __init__(self, bucket: str, service_account_json: str, acl: str = 'private'):
```

Comment:
@ribrea Please add documentation for the parameters

Reply:
I tough it's clear enough, i add param document.
Fixed #4d8dde9